### PR TITLE
Prisma Cloud Compute docs update

### PR DESCRIPTION
**Pull request by BEAdi**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-dc.paloaltonetworks.com/browse/CIAC-9689

## Description
A few sentences describing the overall goals of the pull request's commits.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/PrismaCloudCompute/README.md [hunk 1]
```diff
@@ -2,73 +2,98 @@
...
-6. Click **Save**.
-   * In Cortex XSIAM, once alerts start to come in, a green checkmark appears underneath the Prisma Cloud Compute Collector configuration with the amount of data received.
-7. After Cortex XSIAM begins receiving data from Prisma Cloud Compute, you can use XQL Search to search for specific data using the `prisma_cloud_compute_raw` dataset.
-
-
-**Pay Attention**:
-Timestamp parsing support is available for the **time** field in `%h %d, %Y %H:%M:%S UTC` format (E.g `Oct 14, 2023 09:16:04 UTC`)
+## Overview
+
+This integration lets you import **Palo Alto Networks - Prisma Cloud Compute** alerts into XSIAM
```
Comment (ShirleyDenkberg) [inline Packs/PrismaCloudCompute/README.md:7]:
```suggestion
This integration lets you import **Palo Alto Networks - Prisma Cloud Compute** alerts into Cortex XSIAM.
```

File: Packs/PrismaCloudCompute/README.md [hunk 2]
```diff
@@ -2,73 +2,98 @@
...
+## Use Cases
+
+Manage Prisma Cloud Compute alerts in Cortex XSIAM.
+You can create new playbooks, or extend the default ones, to analyze alerts, assign tasks based on your analysis, and open tickets on other platforms.
+
+## Configure Prisma Cloud Compute
+
+Configure Prisma Cloud Compute to send alerts to Cortex XSIAM by creating an alert profile.
+
+1. Login to your Prisma Cloud Compute console, on new Prisma Cloud versions, go to Runtime Security.
```
Comment (ShirleyDenkberg) [inline Packs/PrismaCloudCompute/README.md:18]:
```suggestion
1. Log in to your Prisma Cloud Compute console. On new Prisma Cloud versions, go to **Runtime Security**.
```

File: Packs/PrismaCloudCompute/README.md [hunk 3]
```diff
@@ -2,73 +2,98 @@
...
+1. Search for **Prisma Cloud Compute**.
+1. Click **Add instance** to create and configure a new integration.
+* **Name**: Name for the integration.
+* **Fetches incidents**: Configures this integration instance to fetch alerts from Prisma Cloud Compute.
+* **Prisma Cloud Compute Console URL**: URL address of your Prisma Cloud Compute console. Copy the address from the alert profile created in Prisma Cloud Compute, or under **Runtime Security** copy the address from **System > Utilities > Path to Console**.
+* **Prisma Cloud Compute Project Name (if applies)**: If using projects in Prisma Cloud Compute, enter the project name here. Copy the project name from the alert profile created in Prisma Cloud Compute.
+* **Trust any certificate (not secure)**: Skips verification of the CA certificate (not recommended).
+* **Use system proxy settings**: Uses the system's proxy settings.
+* **Credentials**: Prisma Cloud Compute login credentials.
+* **Prisma Cloud Compute CA Certificate**: CA Certificate used by Prisma Cloud Compute. Copy the certificate from the alert profile created in Prisma Cloud Compute.
```
Comment (ShirleyDenkberg) [inline Packs/PrismaCloudCompute/README.md:37]:
```suggestion
   * **Name**: Name for the integration.
   * **Fetches incidents**: Configures this integration instance to fetch alerts from Prisma Cloud Compute.
   * **Prisma Cloud Compute Console URL**: URL address of your Prisma Cloud Compute console. Copy the address from the alert profile created in Prisma Cloud Compute, or under **Runtime Security** copy the address from **System > Utilities > Path to Console**.
   * **Prisma Cloud Compute Project Name (if applies)**: If using projects in Prisma Cloud Compute, enter the project name here. Copy the project name from the alert profile created in Prisma Cloud Compute.
   * **Trust any certificate (not secure)**: Skips verification of the CA certificate (not recommended).
   * **Use system proxy settings**: Uses the system's proxy settings.
   * **Credentials**: Prisma Cloud Compute login credentials.
* **Prisma Cloud Compute CA Certificate**: CA Certificate used by Prisma Cloud Compute. Copy the certificate from the alert profile created in Prisma Cloud Compute.
```

File: Packs/PrismaCloudCompute/README.md [hunk 4]
```diff
@@ -2,73 +2,98 @@
...
+5. Click **Done** to save the integration.
+
+
+## Using the integration and scripts
+
+The integration ships with four default playbooks:
+* Prisma Cloud Compute - Audit Alert v3
+* Prisma Cloud Compute - Cloud Discovery Alert
+* Prisma Cloud Compute - Compliance Alert
+* Prisma Cloud Compute - Vulnerability Alert
```
Comment (ShirleyDenkberg) [inline Packs/PrismaCloudCompute/README.md:48]:
```suggestion
* **Prisma Cloud Compute - Audit Alert v3**
* **Prisma Cloud Compute - Cloud Discovery Alert**
* **Prisma Cloud Compute - Compliance Alert**
* **Prisma Cloud Compute - Vulnerability Alert**
```

File: Packs/PrismaCloudCompute/README.md [hunk 5]
```diff
@@ -2,73 +2,98 @@
...
+
+3 of the above playbooks (all except _Audit Alert v3_) contain a single script. The script in each playbook encode the raw JSON alerts into Cortex XSIAM objects that can then be used in the playbooks. The scripts are:
```
Comment (ShirleyDenkberg) [inline Packs/PrismaCloudCompute/README.md:50]:
```suggestion
Three of the above playbooks (all except _Audit Alert v3_) contain a single script. The script in each playbook encode the raw JSON alerts into Cortex XSIAM objects that can then be used in the playbooks. The scripts are:
```

File: Packs/PrismaCloudCompute/README.md [hunk 6]
```diff
@@ -2,73 +2,98 @@
...
+
+* PrismaCloudComputeParseComplianceAlert
+* PrismaCloudComputeParseVulnerabilityAlert
+* PrismaCloudComputeParseCloudDiscoveryAlert
```
Comment (ShirleyDenkberg) [inline Packs/PrismaCloudCompute/README.md:54]:
```suggestion
* **PrismaCloudComputeParseComplianceAlert**
* **PrismaCloudComputeParseVulnerabilityAlert**
* **PrismaCloudComputeParseCloudDiscoveryAlert**
```

File: Packs/PrismaCloudCompute/README.md [hunk 7]
```diff
@@ -2,73 +2,98 @@
...
+
+To better understand how playbooks and scripts interoperate, consider the _Prisma Cloud Compute - Vulnerability Alert_ playbook.
+
+* When the playbook is triggered, a task called **Parse Vulnerability Alert** runs.
+* The task runs the **PrismaCloudComputeParseVulnerabilityAlert** script, which takes the `prismacloudcomputerawalertjson` field of the incident (the raw JSON alert data) as input.
+
+![image](https://raw.githubusercontent.com/demisto/content/f808c78aa6c94a09450879c8702a1b7f023f1d4b/Packs/PrismaCloudCompute/doc_files/prisma_alert_raw_input.png)
+
+
+* Click **outputs** to see how the script transformed the raw JSON input into a XSIAM object.
```
Comment (ShirleyDenkberg) [inline Packs/PrismaCloudCompute/README.md:64]:
```suggestion
* Click **outputs** to see how the script transformed the raw JSON input into a Cortex XSIAM object.
```

File: Packs/PrismaCloudCompute/README.md [hunk 8]
```diff
@@ -2,73 +2,98 @@
...
+
+
+![image](https://raw.githubusercontent.com/demisto/content/f808c78aa6c94a09450879c8702a1b7f023f1d4b/Packs/PrismaCloudCompute/doc_files/prisma_alert_outputs.png)
+
+At this point, you can add tasks that extend the playbook to check and respond to alerts depending on the properties of the Cortex XSIAM object.
+
+### Audit Alert v3 playbook
+This playbook is not similar to the other 3 playbooks. It is a default playbook for parsing and enrichment of Prisma Cloud Compute audit alerts.
```
Comment (ShirleyDenkberg) [inline Packs/PrismaCloudCompute/README.md:72]:
```suggestion
This playbook is not similar to the other three playbooks. It is a default playbook for parsing and enrichment of Prisma Cloud Compute audit alerts.
```

File: Packs/PrismaCloudCompute/README.md [hunk 9]
```diff
@@ -2,73 +2,98 @@
...
+The playbook has the following sections:
 
+Enrichment:
+- Image details
+- Similar container events
+- Owner details
+- Vulnerabilities
+- Compliance details
+- Forensics
+- Defender logs.
```
Comment (ShirleyDenkberg) [inline Packs/PrismaCloudCompute/README.md:83]:
```suggestion
- Defender logs
```

**Comment by ShirleyDenkberg** [comment]
@tomer-pan Doc review completed.

**Review by jbabazadeh — APPROVED** [review]

**Pull request merged**